docs/quickstart/README.md

```diff
@@ -0,0 +1,298 @@
+
+```scala
+  import geotrellis.raster.density.KernelStamper
+
+  def stampPointFeature(tile: MutableArrayTile, tup: (SpatialKey, PointFeature[Double])): MutableArrayTile = {
+    val (spatialKey, pointFeature) = tup
+    val tileExtent = ld.mapTransform(spatialKey)
+    val re = RasterExtent(tileExtent, tile)
+    val result = tile.copy.asInstanceOf[MutableArrayTile]
+    KernelStamper(result, kern).stampKernelDouble(re.mapToGrid(pointFeature.geom), pointFeature.data)
+    result
+  }
+
+  import geotrellis.raster.mapalgebra.local.LocalTileBinaryOp
+
+  object Adder extends LocalTileBinaryOp {
+    def combine(z1: Int, z2: Int) = {
+      if (isNoData(z1)) {
+        z2
+      } else if (isNoData(z2)) {
+        z1
+      } else {
+        z1 + z2
+      }
+    }
+    def combine(r1: Double, r2:Double) = {
+      if (isNoData(r1)) {
+        r2
+      } else if (isNoData(r2)) {
+        r1
+      } else {
+        r1 + r2
+      }
+    }
+  }
+
+  def sumTiles(t1: MutableArrayTile, t2: MutableArrayTile): MutableArrayTile = { 
+    Adder(t1, t2).asInstanceOf[MutableArrayTile]
+  }
+```
+
+Note that we require a custom Adder implementation because the built-in tile summation operator returns `NODATA` if either of the cells being added contain a `NODATA` value themselves.
+
+Creating the desired result is then a matter of the following series of operations on `pointRdd`:
+```scala
+  val tileRdd: RDD[(SpatialKey, Tile)] =
+    pointRdd
+      .flatMap(ptfToSpatialKey)
+      .mapPartitions({ partition =>
+        partition.map { case (spatialKey, pointFeature) =>
+          (spatialKey, (spatialKey, pointFeature))
+        }
+      }, preservesPartitioning = true)
+      .aggregateByKey(ArrayTile.empty(DoubleCellType, ld.tileCols, ld.tileRows))(stampPointFeature, sumTiles)
+      .mapValues{ tile: MutableArrayTile => tile.asInstanceOf[Tile] }
+```
+The `mapPartitions` operation simply applies a transformation to an RDD without triggering any kind of shuffling operation.  Here, it is necessary to make the SpatialKey available to `stampPointFeature` so that it can properly determine the pixel location in the corresponding tile.
```

Comment:
It would be good to call out specifically why `asInstanceOf` is used here, since it is considered Bad and we should point it out as Bad but Necessary.